benchmarks/templates/benchmarks/tutorials/models/deepdive_1.html

```diff
@@ -263,6 +263,73 @@ <h3 class="benefits_heading is-size-3-mobile">Part 8: Putting it All Together </
         </div>
     </div>
 </div>
+
+<div class="box leaderboard-table-component">
+    <h3 class="benefits_heading is-size-3-mobile">Part 9: Model Summary Tools</h3>
+    <p class="benefits_info is-size-5-mobile shorter">
+        Depending on how a model is created, you can use one of the following tools to get summary information about it:
+    </p>
+    <ul class="benefits_list">
+        <li><strong>Keras:</strong> If a model is created with Keras, the built-in <span class="special_format">model.summary()</span>
+            method can be used to display a concise summary of the model, showing the configuration of each layer,
+            including the name, type, output shape, and parameter count.</li>
```

Comment:
```suggestion
```

Brain-Score 2.1 (i.e., 3.11 conversion) will only support PyTorch models, so this part might be obsolete. Could theoretically still be useful to include?